designdocs/provider_boosting_implementation.md

```diff
@@ -255,6 +255,41 @@ pub enum Error<T> {
 }
 ```
 
+### NEW: Events
+
+```rust
+  /// ...
+  /// The target of a staked amount was changed to a new MessageSourceId
+  StakingTargetChanged {
+   /// The account that retargeted the staking amount
+   account: T::AccountId,
+   /// The Provider MSA that the staking amount is taken from
+   from_msa: MessageSourceId,
+   /// The Provider MSA that the staking amount is retargeted to
+   to_msa: MessageSourceId,
+   /// The amount in token that was retargeted
+   amount: BalanceOf<T>,
+  },
+  /// Tokens have been staked on the network for Provider Boosting
+  ProviderBoosted {
```

Comment:
The Provider Boost branch updates the Design Docs; let's keep those updates there instead of causing conflicts.